doc/services/device_mgmt/smp_groups/smp_group_1.rst

```diff
@@ -304,6 +322,9 @@ to continue it in case when it gets broken.
 If library gets request with "off" equal zero it checks stored "sha" within its
 state and if it matches it will respond to update client application with
 offset that it should continue with.
+In case when application supports image post processing, and responses return
+"install_progress" value, after receiving final response with "off" eqal
+to image size, application can keep sending
```

Comment:
`can keep sending` : [wording] subject is missing

Reply:
Will fix.